Lib/test/test_functools.py

```diff
@@ -2696,7 +2696,10 @@ def static_func(arg: int) -> str:
             A().static_func
         ):
             with self.subTest(meth=meth):
-                self.assertEqual(meth.__doc__, 'My function docstring')
+                self.assertEqual(meth.__doc__,
+                                 ('My function docstring'
+                                  if support.HAVE_DOCSTRINGS
+                                  else None))
```

Comment:
Wow, I think this was one of the first tests I ever added to CPython! Sad that it had a bug 😆